koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -58,6 +60,8 @@
 else:
     osbs_flatpak_support = True
 
+BUILD_PARAMS_SCHEMA_PATH = '/usr/share/koji-containerbuild/schemas/build_params.json'
```

Comment:
Could use `pkg_resources.resource_stream` to get this path I think. See how atomic-reactor does it: https://github.com/containerbuildsystem/atomic-reactor/blob/1.6.41/atomic_reactor/util.py#L1269-L1274

Reply:
If I'm understanding it correctly, `pkg_resources.resource_stream` works in atomic-reactor because the json files are packaged under the `atomic_reactor` module (probably somewhere in `/usr/lib*/python*/atomic_reactor/`). Is there a way to use it for files packaged under `/usr/share/`? Or should the schema file for koji-containerbuild be packaged in the same way that atomic-reactor does it?

Reply:
We can stick with /usr/share, no big deal.